vault/external_tests/raft/raft_test.go

```diff
@@ -422,8 +422,7 @@ func TestRaft_SnapshotAPI_RekeyRotate_Backward(t *testing.T) {
 
 			cluster.BarrierKeys = barrierKeys
 			testhelpers.EnsureCoresUnsealed(t, cluster)
-			testhelpers.WaitForActiveNode(t, cluster)
-			activeCore := testhelpers.DeriveActiveCore(t, cluster)
+			activeCore := testhelpers.WaitForActiveNode(t, cluster)
```

Comment:
There could be other tests that might benefit from querying internally. Since `DeriveActiveCore` is essentially doing the API query in the same fashion as the proposed change with `WaitForActiveNode `, we could use that instead.

Reply:
If my analysis of the cause of the failure is correct, then WaitForActiveNode is wrong to use the internal method.  But having thought about it some more I'm no longer confident in my analysis.